### ISSUE TITLE
Cannot read property 'toLowerCase' of undefined on Laravel Mix

### ISSUE THREAD

**Issue by jjjrmy** [tailwindcss/tailwindcss#1295]
Everything is compiling fine with npm run dev but I get the following error when doing npm run prod.
I've narrowed it down to being a Tailwind or postCss issue because it runs with no errors without the postCSS tailwind config.

TypeError: Cannot read property 'toLowerCase' of undefined
    at /css/app.css:3674:1
    at /css/app.css:3675:3

```
const tailwindcss = require('tailwindcss');

mix.sass('src/app.scss', 'public/css').options({
        processCssUrls: false,
        postCss: [ tailwindcss('tailwind.config.js') ],
});
```

**Comment by adamwathan** [comment]
Can you create a GitHub repo that reproduces the issue? I just tried this in a fresh Laravel project and can't reproduce.

**Comment by jjjrmy** [comment]
@adamwathan 
> Can you create a GitHub repo that reproduces the issue? I just tried this in a fresh Laravel project and can't reproduce.

I took everything out of my app.scss except the following and I'm still getting the same error when I run production.

```
@tailwind base;

@tailwind components;

@tailwind utilities;
```

**Comment by jjjrmy** [comment]
**Okay I solved the issue!!**
The extended colors with blank values were causing the error to throw.

```
module.exports = {
  theme: {
    extend: {
      colors: {
        'primary': '#1f81da',
        'secondary': '#f46800',
        'success': '#1d9232',
        'body': '#f4f9fe',
        'main': '#edf6fd',
        'text': '',
        'label': '#495057',
        'border': '#bbb',
        'input': '',
        'muted': '',
      },
    }
  },
}
```

**Comment by shipstar** [comment]
I just ran into this issue as well and spent a decent amount of time debugging it. I'm not using Laravel - just webpack + postcss - so I overlooked this issue initially.

From a developer experience perspective, is this something Tailwind or PostCSS could detect and provide a more helpful error message? It would have saved me close to an hour if I had gotten a "custom colors cannot be blank" error message instead of the `toLowerCase` one mentioned above.

I'm not sure if this is something that would be handled by Tailwind or PostCSS, but I'm happy to look into it if someone can point me in the right direction.

Thanks!
